### ISSUE TITLE
unable to create ALIAS records in Route53

### ISSUE THREAD

**Issue by varunchandak** [wallix/awless#210]
I am trying to create/update records in route53 with ELB endpoints but I am unable to create ALIAS records.

Also, there is no ALIAS option in the help as well.

```
type........ The DNS record type (A | AAAA | CNAME | MX | NAPTR | NS | PTR | SOA | SPF | SRV | TXT)
```

**Comment by simcap** [comment]
Indeed, this would need to be added to our create/update record code `github.com/wallix/awless/aws/spec/record.go`. 

Looking at the AWS APIs, it is indeed cumbersome to use so it needs to be fitted correctly in `awless` with the goal of making much simpler, as always. 

At the moment we will not have the resource to do that.
